airbyte-cdk/python/unit_tests/sources/declarative/test_factory.py

```diff
@@ -807,3 +808,8 @@ def test_validate_types_nested_in_list():
 def test_unpack(test_name, input_type, expected_unpacked_types):
     actual_unpacked_types = DeclarativeComponentFactory.unpack(input_type)
     assert actual_unpacked_types == expected_unpacked_types
+
+
+def test_complete_schema():
```

Comment:
do we want to keep this test?

Reply:
nope, this is just for my convenience testing, i'll remove it if we're just gonna generate the json file and put it into the repo